extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -189,6 +189,7 @@ class AmpLightbox extends AMP.BaseElement {
     });
 
     if (this.isScrollable_) {
+      this.element./*OK*/scrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
sorry the code link is mispositioned because of recent change in that file. updated the link: https://github.com/ampproject/amphtml/blob/master/extensions/amp-lightbox/0.1/amp-lightbox.js#L148